components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md

```diff
@@ -125,19 +135,19 @@ This page explains migrating the existing A2HA data to the newly deployed Chef A
     sudo chef-automate backup restore /mnt/automate_backups/backups/20210622065515/ --patch-config current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight
     ```
 
-1. After the restore is successfully executed, you will see the below message:
+2. After the restore is successfully executed, you will see the below message:
```

Comment:
I don't think you should change the numbers, as in MD format it auto-increments, example:
```
1. First
1. Second
1. Third
```
1. First
1. Second
1. Third

Reply:
Have fixed it.